.github/workflows/commitlint-autofix.yml

```
@@ -0,0 +1,198 @@
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const prNumber = ${{ steps.get-pr.outputs.pr_number }};
+            const commitCount = ${{ steps.count-commits.outputs.commit_count }};
+            const prTitle = `${{ steps.get-pr.outputs.pr_title }}`;
+
+            let message;
+            if (commitCount > 1) {
+              message = `### Commit Messages Auto-Fixed
+
+The commitlint check failed because one or more commit messages didn't follow [Conventional Commits](https://www.conventionalcommits.org/) format.
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)